butane_codegen/src/lib.rs

```diff
@@ -167,3 +167,39 @@ fn migrations_dir() -> PathBuf {
     dir.push("migrations");
     dir
 }
+
+#[proc_macro_derive(ButaneJson)]
+pub fn derive_butane_json(input: TokenStream) -> TokenStream {
+    let ast = syn::parse_macro_input!(input as syn::DeriveInput);
+    let struct_name = &ast.ident;
+    let s = format!(
```

Comment:
Any reason not to use the `quote!` macro (as the other proc macros in this crate do) rather than parsing a string? 
https://crates.io/crates/quote